rebasehelper/tests/test_specfile.py

```diff
@@ -43,6 +44,7 @@ class TestSpecFile(BaseTest):
     SOURCE_4 = 'file.txt.bz2'
     SOURCE_5 = 'documentation.tar.xz'
     SOURCE_6 = 'misc.zip'
+    SOURCE_7 = 'https://files.pythonhosted.org/packages/source/p/positional/positional-1.1.0.tar.gz'
```

Comment:
This should refer to a local file, at least that's what `TestSpecFile.test_get_sources` method expects.